src/api/integrations/chatbot/typebot/services/typebot.service.ts

```
@@ -40,193 +42,127 @@ export class TypebotService extends BaseChatbotService<TypebotModel, any> {
+  /**
+   * Create a new TypeBot session with prefilled variables
+   */
+  public async createNewSession(instance: Instance, data: any) {
+    if (data.remoteJid === 'status@broadcast') return;
     const id = Math.floor(Math.random() * 10000000000).toString();
 
     try {
-      const version = this.configService?.get<Typebot>('TYPEBOT').API_VERSION;
+      const version = this.configService.get<Typebot>('TYPEBOT').API_VERSION;
       let url: string;
       let reqData: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)